DEVELOPING.md

```diff
@@ -0,0 +1,129 @@
+    protected DatabaseWrapper createContainerWrapper() throws Exception {      
+        return new DockerDatabaseWrapper(new FirebirdContainer(
+                DockerImageName.parse(getImageName()).withTag(getVersion()))
+                .withDatabaseName(getCatalog())
+                .withUsername(getUsername())
+                .withPassword(getPassword()),
+                this
+        );
+    }
+}
+```
+
+Additionally, we need to add the test dependency for the DB we are adding support for in the `pom.xml` file from **liquibase-extension-testing** module:
+```xml
+<dependency>
+    <groupId>org.firebirdsql</groupId>
+    <artifactId>firebird-testcontainers-java</artifactId>
+    <version>1.2.0</version>
+</dependency>
+```
+
+#### Controlling which systems are tested against
+
+Even though systems are able to be automatically brought up, tests against databases are slow. Therefore, exactly which test systems should be tested against is controllable by you.
+By default, Liquibase will only test against in-memory databases (h2, hsqldb, sqlite) because they are fast. Any tests which ask for other systems will automatically be marked as "ignored".
+
+To change which tests run and which are ignored, set the `liquibase.sdk.testSystem.test` attribute to be a comma-separated list. This can be most easily set in your liquibase.sdk.local.yml file (see below).
+
+The `liquibase.sdk.testSystem.test` setting can also specify specific settings for the test systems you wish to test against. 
+For example, if you have `liquibase.sdk.testSystem.test=h2?version=1.4.200,mysql`, when your tests call `getTestSystem("h2")` it will get an h2 version 1.4.200 system. You can also specify profiles (see below) as well, like `liquibase.sdk.testSystem.test=mssql:case-sensitive,mysql`    
+
+#### Starting test systems independently from tests
+
+Often times you need to have a database you can manually run liquibase against outside of your automated tests. 
+The `liquibase-extension-testing.jar` and corresponding `liquibase-extension-testing-deps.jar` files add a `liquibase sdk system` command group to the Liquibase CLI which allow you to start and stop the same containers used by your test.
+By adding those jars to your LIQUIBASE_HOME/lib directory, you can run `liquibase sdk system up --name mysql` to bring up a mysql system, where the `name` argument is the name of the test system you want to start.
+
+To shut down the instances started this way, run `liquibase sdk system down --name mysql`
+
+#### Local configuration
+
+The liquibase.sdk.yaml file contains the default configuration, but there are times you may want to change the default behavior. 
+By creating a `liquibase.sdk.local.yml` file in the [same directory](liquibase-extension-testing/src/main/resources) as the liquibase.sdk.yaml file, you can add just the settings you want to override.
+NOTE: gitignore is set to not allow the file to be committed because it's designed to be specific to you.
```

Comment:
Edit at line 94:

"NOTE: The `liquibase.sdk.local.yml` is specific to your development environment and gitignore is configured to disallow the  file from being committed."

Reply:
Done.